actors/miner/src/testing.rs

```diff
@@ -79,7 +80,7 @@ pub fn check_state_invariants<BS: Blockstore>(
                     miner_summary.deals.insert(
                         deal,
                         DealSummary {
-                            sector_start: sector.activation,
+                            sector_start: max(sector.power_base_epoch, sector.activation),
```

Comment:
I'm not confident this change is valid, but might need to explore more to understand it.